probe.go

```diff
@@ -36,6 +36,10 @@ func handleProbe(metrics collector.Metrics, scrapers []collector.Scraper, logger
 			http.Error(w, "target is required", http.StatusBadRequest)
 			return
 		}
+		network := params.Get("network")
```

Comment:
Thinking about this a bit, what if instead of having a separate `network` param, we just check for `strings.HasPrefix(target, "unix://")`?

Reply:
It sounds fine. Or as another option, how do you think using separate parameters `host` for TCP and `socket` for UDP (and prefer the former if both exist) instead of using `target` for both networks? Two parameter names conform to the MySQL config.

Reply:
No, I think we can keep this as a single param and check for `unix://` and remove that prefix.

For example we do this in other parts of Prometheus, like connecting to the docker socket.

`unix:///var/run/docker.sock` becomes `/var/run/docker.sock`.